src/components/Restaurants/WriteReviewPage.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import StarRatingInput from './StarRatingInput'; 
+import './WriteReviewPage.css';
+
+function WriteReviewPage() {
+    const [review, setReview] = useState("");  
+    const [submitting, setSubmitting] = useState(false);
+    const [submitted, setSubmitted] = useState(false);
+    const [rating, setRating] = useState(0);
+    const [selectedTags, setSelectedTags] = useState([]);
+
+    const handleInputChange = (event) => {
+        setReview(event.target.value);
+    };
+
+    const handleSubmit = (event) => {
+        event.preventDefault(); 
+        if (review.trim() === "" || rating === 0) {
+            alert("Please enter a review and select a rating before submitting.");
+            return;
+        }
+
+        setSubmitting(true);
+
+        setTimeout(() => {
+            console.log('Review Submitted:', review); 
+            setSubmitting(false);
+            setSubmitted(true);
+            setReview(""); 
+            setRating(0);
+            setSelectedTags([]);
+        }, 1000); 
```

Comment:
Add `navigate('/detail/write-review/success');` in this function

Need to import and instantiate like
```
import { useNavigate } from 'react-router-dom';
...
const navigate = useNavigate();
```